database/knexfile.js

```diff
@@ -6,6 +6,10 @@ module.exports = {
   searchPath: [env.PGSCHEMA, 'public'],
   migrations: {
     tableName: 'knex_migrations',
-    schemaName: env.PGSCHEMA
-  }
+    schemaName: env.PGSCHEMA,
+  },
+  pool: {
+    min: 1,
+    max: 100,
```

Comment:
@k1ch  are these the default values for knex? Should we make these load from environment variables, and then if not specified fallback to a default value of 1 and 100?

Reply:
The default values are `min: 2` and `max: 10`. Good idea I will introduce ENV variable to easily change configuration.